R/community.R

```diff
@@ -2041,8 +2039,16 @@ cluster_label_prop0 <- function(
   if (!is.null(initial)) initial <- as.numeric(initial)
   if (!is.null(fixed)) fixed <- as.logical(fixed)
 
-  directed <- switch(igraph.match.arg(mode), "out" = TRUE, "in" = TRUE, "all" = FALSE)
-  mode <- switch(igraph.match.arg(mode), "out" = 1L, "in" = 2L, "all" = 3L)
+  directed <- switch(igraph.match.arg(mode),
+    "out" = TRUE,
```

Comment:
```suggestion
    out = TRUE,
```

Not necesarily to be done in this PR but this week lintr told me not to unnecessarily quote arguments, always for `switch()` calls where I had done so, so now I'm passing it along :joy_cat: